games-engines/gargoyle/gargoyle-2011.1.ebuild

```diff
@@ -69,7 +68,7 @@ src_compile() {
 		-sC++="$(tc-getCXX) ${CXXFLAGS}" \
 		-sCXX="$(tc-getCXX) ${CXXFLAGS}" \
 		-sC++FLAGS="" \
-		-sGARGLKINI="${GAMES_SYSCONFDIR}/garglk.ini" \
+		-sGARGLKINI="/etc/garglk.ini" \
```

Comment:
This changes the location of a file on disk, from /etc/games to /etc; this means you'll need to bump the revision on the ebuild.

Reply:
Oops. Fixed.